fe1-web/src/core/keypair/index.ts

```diff
@@ -0,0 +1,7 @@
+import { addReducer } from '../redux/Manage';
+import keyPairReducer from './Reducer';
+
+export * from './Reducer';
```

Comment:
Why did you rename the reducer ? It being called simply 'reducer' feels like it's generic.